Formula/opencsg.rb

```diff
@@ -0,0 +1,74 @@
+class Opencsg < Formula
+  desc "The CSG rendering library"
+  homepage "http://www.opencsg.org"
+  url "http://www.opencsg.org/OpenCSG-1.4.0.tar.gz"
+  sha256 "ecb46be54cfb8a338d2a9b62dec90ec8da6c769078c076f58147d4a6ba1c878d"
+
+  depends_on "qt5" => :build
+  depends_on "glew"
+
+  def install
+    system "qmake", "-r", "INSTALLDIR=#{prefix}",
+      "INCLUDEPATH+=#{HOMEBREW_PREFIX}/include",
+      "LIBS+=-L#{HOMEBREW_PREFIX}/lib -lGLEW"
+
+    system "make", "install"
+  end
+
+  def patches
```

Comment:
I'll update the patch. In terms of patch location, it's a bit of a chicken & egg problem - will formula-patches accept the patch prior to the formula, or do I just pull it from a private location in the interim?

I'm submitting the patch upstream once I'm confident that it will work with Homebrew, as the upstream author tends to take time to review such patches.